hello.md

```diff
@@ -1,6 +1,8 @@
-# hello
-My name is Cow.
 
-I like to play with RHesidents.
+My name is <---Insert name--->.
```

Comment:
Did you accidentally leave this line in or is it there because you want to siam the merge conflict?

Reply:
I ll try one more time then